src/PythonBindings/CharmCompatibility.cpp

```diff
@@ -77,10 +79,18 @@ std::string get_paths() noexcept { return "Not supported in python."; }
  * These functions are not intended to be called.
  */
 
-PUP::able::~able() {
+// clang-tidy: can't use `= default` to define a trivial destructor
+PUP::able::~able() {  // NOLINT
```

Comment:
try doing what clang-tidy say and see what breaks?

Reply:
it didn't expect this to work but it did 😄 thanks 👍